bower.json

```diff
@@ -24,10 +24,10 @@
     "preview.png"
   ],
   "dependencies": {
-    "polymer": "polymer/polymer#^1.1.0"
+    "polymer": "polymer/polymer#1.9 - 2"
   },
   "resolutions": {
-    "polymer": "^1.1.0",
+    "polymer": "~1.9",
     "webcomponentsjs": "^0.7.18"
   }
```

Comment:
Polymer gives me warnings that the above is an "unneccessary resolution" and installs Polymer 2 anyway:

![2017-07-25 10_36_51-command prompt](https://user-images.githubusercontent.com/566463/28563131-3d8821d0-7125-11e7-8eec-0c0c8ad32c35.png)